js/script.js

```diff
@@ -63,3 +54,65 @@ function clean() {
     xNumber = null;
     yNumber = '';
 }
+
+function decimalEntry(dec) {
+    if (!output.value.includes(dec)) {
+        output.value += dec;
+    }
+    
+    if (operatorClicked) {
+        output.value = '0' + dec;
+        operatorClicked = false;
+    }
+}
+
+function numberNegative() {
+    output.value = -(output.value);
+}
+
+function rootSquare() {
+    if (output.value) {
+        output.value = Math.sqrt(output.value);
+        xNumber = output.value;
+        operatorClicked = true;
+    }
+}
+
+function initialize () {
+    const globalNumbers = document.querySelectorAll('.action-bar__number');
+    for (let i = 0; i < globalNumbers.length; i++) {
+        globalNumbers[i].addEventListener('click', event => 
+        onNumberClick(event.target.defaultValue)
+        );
+    }
+
+    const operators = document.querySelectorAll('.action-bar__operator');
+    for (let i = 0; i < operators.length; i++) {
+        operators[i].addEventListener('click', event =>
+        operation(event.target.defaultValue)
+        );
+    }
+
+    const decimal = document.getElementById('dec');
+    decimal.addEventListener('click', event => 
+    decimalEntry(event.target.defaultValue)
+    );
```

Comment:
Once again: fix all indentation problems. In this example, you have absolutely flat structure. Here's the correct indentation:

```
decimal.addEventListener('click', event => 
    decimalEntry(event.target.defaultValue)
);
```

You can beautify your code here (just copy and paste the whole file): https://beautifier.io/. It will help to understand correct indentation.